zubhub_frontend/zubhub/src/views/create_project/CreateProject.jsx

```diff
@@ -430,11 +434,11 @@ const SelectModeUI = ({ setMode }) => {
             <Typography>If you worked on the project with other creators </Typography>
           </div>
         </div>
+        {isError && <div className={classes.projectSelectError}>You need to select project type.</div>}
         <CustomButton
           onClick={handleCreate}
           primaryButtonStyle
           style={{ marginTop: 40, alignSelf: 'center' }}
-          disabled={mode.length === 0}
```

Comment:
Can you keep the button disabled? Some big-headed users might decide to click Create project when they haven't selected a mode. We generally want to limit them to that point until they choose one

Reply:
Hey, with these changes, it is still not possible to create a project without selecting a mode. Do you suggest showing the error and disabling the button at the same time?

Reply:
Alright. I think its fine then